### ISSUE TITLE
[android]Switch back to system keyboard

### ISSUE THREAD

**Issue by carnivion** [nesbox/TIC-80#679]
Add button or config to switch to system keyboard on android, or add Dvorak layout for soft-keyboard(which I made one as ref).
![temp1](https://user-images.githubusercontent.com/4220495/44151126-303f4ca8-a0d4-11e8-8bbf-39379eb1c1b5.png)

**Comment by carlsmith** [comment]
I was playing around with a new keyboard layout that uses one less column and is half a row shorter, so the keys can be a bit bigger.

*Note: Ignore the style of keyboard. I used a layout tool, and that's just how the default styles look*.

The top priority was preserving the location of the letter and number keys.

I tried to avoid moving keys too far from where people are used to looking for them, and to at least cluster similar keys (quotes, slashes, brackets) when they have been relocated, so you quickly get used to roughly where they are.

All the characters that are normally on the number keys are moved up one row (so you don't need to use Shift for those).

Note: Shift-Dot inputs a colon, and Shift-Comma inputs a semi-colon (but the legends looked confusing in the tool I used).

![keyboard-layout](https://user-images.githubusercontent.com/7561668/45341830-d7ca9d00-b593-11e8-94af-2b4e8854758b.png)
